website/.spelling

```diff
@@ -778,6 +778,10 @@ APPROX_COUNT_DISTINCT_DS_THETA
 APPROX_QUANTILE_DS
 DS_QUANTILES_SKETCH
 APPROX_QUANTILE_FIXED_BUCKETS
+# Operators
+pivoted
+UNPIVOT
```

Comment:
should PIVOT be included here as well ?

Reply:
It didn't fail any spelling tests, so I don't think so.